index.html

```diff
@@ -91,14 +94,14 @@ <h2>Messages</h2>
 
       <footer>
         <section id="connect">
-            <a href="mailto:keriwest88@gmail.com?subject=Mail from Website" target="_blank">
-              <img src="img/email-icon.png"></a>
+          <a href="mailto:keriwest88@gmail.com?subject=MailfromWebsite" target="_blank">
```

Comment:
Spaces would actually be good, it's just preferable to make sure they're URL encoded.